src/Renci.SshNet/Sftp/SftpSession.cs

```diff
@@ -444,6 +495,24 @@ public byte[] EndOpen(SftpOpenAsyncResult asyncResult)
             }
         }
 
+#if FEATURE_TAP
+        public async Task<byte[]> OpenAsync(string path, Flags flags, CancellationToken cancellationToken)
```

Comment:
Please rename to **RequestOpenAsync** and move it next to the **RequestOpen** method.